packages/frontapp/src/app/globalUI/Status/index.tsx

```diff
@@ -10,28 +10,7 @@ export const StyledBookStatusWrapper = styled.div<IStyledBookStatusProps>`
   @media (max-width: ${dimensions.phone_width}) {
     width: fit-content;
     padding: 0 4px;
-    text-align: center;
-    border-width: 1px;
-    border-style: solid;
     border-radius: 4px;
-    border-color: ${(props) => {
-      switch (props.status) {
-        case StatusTypes.FREE:
-          return colors.free_book;
-        case StatusTypes.BUSY:
```

Comment:
are you sure that book status will not be different from figma design, if u will delete all border colors?

Reply:
on all cards there are no borders for the status